tests/tensorflow/sota_checkpoints_eval.json

```diff
@@ -203,7 +218,8 @@
                     "compression_description": "INT8",
                     "mean_value": "[103.939,116.779,123.68]",
                     "diff_fp32_min": -1,
-                    "diff_fp32_max": 0.1
+                    "diff_fp32_max": 0.1,
+                    "diff_target_min": -0.17
```

Comment:
What is the reason for these changes?

Reply:
It was because of our discussion for the pr in teams, which is adjusting the tolerance for resnet int8.

Reply:
I think we had some misunderstanding. You need to update metrics here and readme for new checkpoints only.

Reply:
Yes, updated!